frontend/src/styles.scss

```
@@ -101,3 +101,142 @@ body {
+        left: 0 !important;
+        right: 0 !important;
+        top: auto !important;
+        max-width: 100vw !important;
+        width: 100vw !important;
+        margin: 0 !important;
+        animation: mobileBottomSheetSlideUp 260ms cubic-bezier(0.2, 0.8, 0.2, 1);
+    }
+
+    .mobile-bottom-sheet-dialog .mat-mdc-dialog-container {
+        --mdc-dialog-container-shape: 16px 16px 0 0;
+        max-height: 85vh;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)